tsconfig.json

```diff
@@ -1,38 +1,27 @@
 {
+  "extends": "./tsconfig.base.json",
   "compilerOptions": {
-    "jsx": "react",
```

Comment:
These options are moved to `tsconfig.base.json` so it can then be reused in both the main app and Cypress' `tsconfig.json`